<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalcMainWindow</class>
 <widget class="QMainWindow" name="CalcMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>有理数计算器</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow,QLabel,QMenuBar
{
	border:none;
	background-color: rgb(240, 244, 249);
}

QPushButton
{
	border:1px solid rgb(227, 229, 234);
	background-color: rgb(247, 249, 252);
}

QPushButton:hover
{
	border:2px solid rgb(227, 229, 234);
	background-color: rgb(244, 246, 250);
}

QPushButton:pressed
{
	border:1px solid rgb(227, 229, 234);
	background-color: rgb(242, 244, 250);
}

QPushButton#btnZero,#btnOne,#btnTwo,#btnThree,#btnFive,#btnFour,#btnSix,#btnSeven,#btnEight,#btnNine,#btnPoint,#btnPNchange
{
	border:1px solid rgb(227, 229, 234);
	background-color: rgb(255,255,255);
}

QPushButton#btnZero:hover,#btnOne:hover,#btnTwo:hover,#btnThree:hover,#btnFive:hover,#btnFour:hover,#btnSix:hover,#btnSeven:hover,#btnEight:hover,#btnNine:hover,#btnPoint:hover,#btnPNchange:hover
{
	border:2px solid rgb(227, 229, 234);
	background-color: rgb(251, 252, 253);
}

QPushButton#btnZero:pressed,#btnOne:pressed,#btnTwo:pressed,#btnThree:pressed,#btnFive:pressed,#btnFour:pressed,#btnSix:pressed,#btnSeven:pressed,#btnEight:pressed,#btnNine:pressed,#btnPoint:pressed,#btnPNchange:pressed
{
	border:1px solid rgb(227, 229, 234);
	background-color: rgb(248, 249, 252);
}

QPushButton#btnEqual{
	color: rgb(255,255,255);
	background-color: rgb(0, 103, 192);
}

QPushButton#btnEqual:hover{
	color: rgb(255,255,255);
	background-color: rgb(24, 117, 197);
}

QPushButton#btnEqual:pressed{
	color: rgb(255,255,255);
	background-color: rgb(48, 131, 203);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="indent">
       <number>8</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>65</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>65</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="indent">
       <number>8</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>14</pointsize>
        <fontweight>Light</fontweight>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="indent">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="5" column="3">
       <widget class="QPushButton" name="btnEqual">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>81</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnComb">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>81</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>nCm</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="btnTwo">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>81</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="btnFractionChar">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>81</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>|(分数线)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnFractorial">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>81</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>!</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btnBackspace">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>81</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Backspace</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="btnZero">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>81</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="btnMultiply">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>81</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="btnPlus">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>81</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btnFive">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>81</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="btnNine">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>81</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="btnPNchange">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>81</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>+/-</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="btnThree">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>81</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnCLear">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>81</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>CE</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="btnOne">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>81</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="btnPoint">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>81</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btnFour">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>81</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="btnSix">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>81</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnClearAll">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>81</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btnEight">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>81</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnPower">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>81</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>^</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btnSeven">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>81</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="btnMinus">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>81</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnDiv">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>134</width>
          <height>81</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>÷</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>362</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>设置(&amp;S)</string>
    </property>
    <addaction name="actDisplayDecimal"/>
    <addaction name="actSetPreciese"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="actExit">
   <property name="text">
    <string>退出(&amp;Q)</string>
   </property>
  </action>
  <action name="actDisplayDecimal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>显示小数结果(&amp;D)</string>
   </property>
  </action>
  <action name="actSetPreciese">
   <property name="text">
    <string>设置小数精度(&amp;P)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
